pkg/cmd/run.go

```diff
@@ -223,7 +223,7 @@ doppler run --mount secrets.json -- cat secrets.json`,
 			// remove any reserved keys from secrets
 			reservedKeys := []string{"PATH", "PS1", "HOME"}
 			for _, reservedKey := range reservedKeys {
-				if _, found := dopplerSecrets[reservedKey]; found == true {
```

Comment:
nit: typo in commit message: `chore: Omit comparaison with boolean constant`

Reply:
Thanks!